src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,108 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTORY = 0.75f;
+    private static final float LOAD_FACTORY = DEFAULT_LOAD_FACTORY;
```

Comment:
Why do you need two LOAD_FACTORY constants?

Reply:
Because AI mentor said that I used hardcode. 
AI-mentor:
Any magic numbers should be constants. Your code should be easy to read. Move all hardcoded values to constant fields and give them informative names. For example: constants can be defaultCapacity, loadFactor...